pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -143,6 +143,7 @@ func checkDirectIOFlag(path string) bool {
 	if err != nil && !os.IsNotExist(err) {
 		return false
 	}
+	// #nosec No need to check Close() errors - no writing just checking O_DIRECT flag
```

Comment:
Lets not use comments like these. Please log it.

Reply:
It's a missed left over.

Reply:
Fixed